grails-app/services/io/xh/toolbox/user/RoleService.groovy

```diff
@@ -15,21 +15,10 @@ class RoleService extends DefaultRoleService {
 
     void init() {
         super.init()
-        assignLocalAdminRolesIfNeeded()
     }
```

Comment:
Let's remove the init() method here entirely, since we have no unique work to do in it other than call super